files/management/commands/consolidate_language_duplicates.py

```
@@ -0,0 +1,135 @@
+                    FROM files_language
+                    GROUP BY code
+                    HAVING COUNT(*) > 1
+                """)
+                duplicates = cursor.fetchall()
+
+                if duplicates:
+                    for code, count, ids, titles in duplicates:
+                        self.stdout.write(f"\nCode '{code}' appears {count} times:")
+                        id_list = ids.split(',')
+                        title_list = titles.split(',')
+                        for lang_id, title in zip(id_list, title_list):
```

Ruff (0.13.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)